library/teams.cfc

```diff
@@ -71,4 +71,71 @@
 		<cfreturn getTeam.teamName>
 		
  </cffunction>
+
+ <cffunction name="addTeam" access="remote" returntype="struct" hint="Add a new team to the database">
```

Comment:
All the changes in this file are chatGPT @Jwwason @Jonan-Doan hahahahahahahahahahaha